cpp/log/cert.cc

```diff
@@ -233,8 +262,8 @@ string Cert::PrintTime(ASN1_TIME* when) {
 }
 
 
-Cert::Status Cert::IsIdenticalTo(const Cert& other) const {
-  return X509_cmp(x509_, other.x509_) == 0 ? TRUE : FALSE;
+StatusOr<bool> Cert::IsIdenticalTo(const Cert& other) const {
+  return StatusOr<bool>(X509_cmp(x509_, other.x509_) == 0 ? true : false);
```

Comment:
You normally don't need to construct a temporary, `StatusOr<bool>` has two non-explicit constructors that takes a `util::Status` or `bool`. Stripping out that "stutter" should make things more readable (here, and for every other `return` statements in the file).